src/components/filter-selector/FilterSelector.tsx

```diff
@@ -30,7 +33,7 @@ interface FilterSelectorProps<T> extends Omit<MenuProps, 'open'> {
   valueField?: keyof T
 }
 
-const FilterSelector = <T,>({
+const FilterSelector = <T extends HasId>({
```

Comment:
Pick<CategoryNameInterface, '_id'> instead of creating HasId

Reply:
done